src/test/scala/sigmastate/serialization/DataJsonEncoderSpecification.scala

```diff
@@ -0,0 +1,128 @@
+package sigmastate.serialization
+
+
+import java.math.BigInteger
+
+import org.scalacheck.Arbitrary._
+import scalan.RType
+import sigmastate.SCollection.SByteArray
+import sigmastate.SType.AnyOps
+import sigmastate.Values.SigmaBoolean
+import sigmastate._
+import sigmastate.eval.Extensions._
+import sigmastate.eval.{Evaluation, _}
+import sigmastate.interpreter.CryptoConstants.EcPointType
+import special.sigma.{AvlTree, Box}
+
+class DataJsonEncoderSpecification extends SerializationSpecification {
```

Comment:
@gostkin A negative test for tuples with more than 2 items would be nice (check out http://github.com/ScorexFoundation/sigmastate-interpreter/blob/cf570014ba16bad555a73d71d539568a4cf341a1/src/test/scala/sigmastate/serialization/ConcreteCollectionSerializerSpecification.scala#L58-L58 for example)